server/services/user.test.js

```diff
@@ -1,6 +1,46 @@
 const UserService = require('./user');
 
 describe('userService', () => {
+  describe('getUser successfully (200)', () => {
+    // GIVEN
+    const user = {
+      email: 'testemail123@gmail.com',
+      password: 'testing123',
+    };
+    const userDao = {
+      getUser: (_) => user,
```

Comment:
What's the purpose of the `_` here?

Reply:
Just a naming convention specifying this parameter is required but won't be used. There's no other purposes.